javascript/002-objetos-js-em-detalhe.md

```diff
@@ -3,91 +3,91 @@
 * **Artigo original**: [JavaScript Objects in Detail](http://javascriptissexy.com/javascript-objects-in-detail/)
 * **Tradução**: [Eric Douglas](https://github.com/ericdouglas)
 
-O núcleo mais utilizado e mais básico do JavaScript é o Object. O JavaScript tem um tipo complexo de dado, o tipo Object, e tem cinco tipos de dados simples: Number, String, Boolean, Undefined e Null. Note que estes tipos simples (primitivos) de dados são imutáveis, eles não podem ser alterados, enquanto os objetos podem.
+O tipo de dado central, o mais fundamental e freqüentemente utilizado, do JavaScript é o tipo Object. O JavaScript tem 1 tipo de dado complexo, o tipo Object, e tem 6 tipos de dados simples: Boolean, Number, String, Symbol, undefined e null. Note que estes tipos de dados simples (primitivos) são imutáveis (não podem ser alterados), enquanto os objetos são mutáveis (podem ser alterados).
```

Comment:
Ao comparar com o texto original, deparei-me com algumas inclusões que não constam por lá. Por exemplo:

> ... e cada nome de propriedade deve ser único e pode ser uma string ou um número.

Daí pensei que se fosse necessário, não haveria problema algum de acrescentar algo que ajudasse a elucidar melhor o conteúdo original. E até mesmo tornar o texto + up-to-date com o ES6.

Mas se você considera mais importante ser fiel ao máximo possível ao original, mesmo em detrimento de um entendimento melhor e/ou deixando o conteúdo defasado, removê-los-ei todos.  :P